lang/cs/Org.Apache.REEF.Client/Avro/YARN/AvroYarnJobSubmissionParameters.cs

```diff
@@ -60,6 +60,12 @@ public static string Schema
         public string jobSubmissionDirectoryPrefix { get; set; }
 
         /// <summary>
+        /// Gets or sets file system url
```

Comment:
This is store URL, example is like adl://xxxx. In ADL cluster, fs.defaultFS is not set in the core-site.xml. REEF has to set it before using the FileSystem. Also when a user submit a job, he can specify store URL that can be different from cluster default. Therefore we need a way to allow it is passed in.